src/main/resources/swagger/api-docs.yaml

```diff
@@ -4750,9 +4748,10 @@ components:
             type: string
   securitySchemes:
     oidc:
-      type: openIdConnect
-      openIdConnectUrl: OPEN_ID_CONNECT_URL
+      type: oauth2
       x-tokenName: id_token
-    bearerAuth:
-      type: http
-      scheme: bearer
\ No newline at end of file
+      flows:
+        authorizationCode:
+          authorizationUrl: OAUTH_AUTHORIZATION_URL
+          scopes: {}
+          tokenUrl: OAUTH_TOKEN_URL
```

Comment:
replacements for `OAUTH_AUTHORIZATION_URL` and `OAUTH_TOKEN_URL` were implemented in [workbenchLibs](https://github.com/broadinstitute/workbench-libs/pull/1684)